src/jobs/withdrawalProcessingJob.ts

```
@@ -110,27 +110,43 @@ export async function startWithdrawalProcessingConsumer(): Promise<void> {
-            recipient,
-          );
+          // Try all providers for this currency, failover if needed
+          let provider, result, usedProviderId;
+          try {
+            provider = await router.getProvider(currency);
+            usedProviderId = provider.constructor?.name || "unknown";
+            result = await provider.disburseFunds(
+              amount,
+              currency,
+              recipient,
+            );
```

ESLint

[error] Replace `⏎··············amount,⏎··············currency,⏎··············recipient,⏎············` with `amount,·currency,·recipient`

(prettier/prettier)